cmake/modules/FindReadline.cmake

```diff
@@ -4,13 +4,16 @@ find_path(READLINE_INCLUDE_DIRS readline/readline.h
 	HINTS ENV READLINE_DIR
 	PATH_SUFFIXES include
 )
-find_library(READLINE_LIBRARIES_FOUND
+find_library(READLINE_LIBRARIES
 	NAMES readline
 	HINTS ENV READLINE_DIR
 )
-if(READLINE_LIBRARIES_FOUND)
-	set(READLINE_LIBRARIES readline)
-endif(READLINE_LIBRARIES_FOUND)
+find_library(TINFO_LIBRARIES
+  NAMES tinfo termcap ncursesw ncurses cursesw curses
```

Comment:
`find_library` finds one of available library from the list. It seems to select wrong library if tinfo is not existing. (libtermcap will be selected instead